library/oss/localstack/providers.cue

```diff
@@ -1,12 +1,28 @@
 providers: {
 	"namespacelabs.dev/foundation/library/storage/s3:Bucket": {
-		initializedWith: "namespacelabs.dev/foundation/library/oss/localstack/prepare"
+		initializedWith: "namespacelabs.dev/foundation/library/oss/localstack/prepare/bucket"
 
-		resources: {
-			server: {
-				class:  "namespacelabs.dev/foundation/library/runtime:Server"
-				intent: "namespacelabs.dev/foundation/library/oss/localstack/server"
-			}
+		inputs: {
+			// TODO define default
+			"server": "namespacelabs.dev/foundation/library/oss/localstack:Server"
 		}
 	}
+	"namespacelabs.dev/foundation/library/oss/localstack:Server": {
```

Comment:
I suggest we use "Cluster" to keep some parity with postgres. Although it's not 100% accurate, that's kind of what we're trying to do here.

Reply:
I renamed the Server to Cluster and added default with last commit